internal/backends/winit/winitwindowadapter.rs

```diff
@@ -459,6 +461,92 @@ impl WindowAdapter for WinitWindowAdapter {
         self.size.get()
     }
 
+    fn is_window_background_blurred(&self) -> bool {
+        self.blurred.get()
+    }
+
+    fn set_window_background_blurred(&self, blur: bool) {
+        let _old = self.blurred.replace(blur);
+
+        #[cfg(target_os = "macos")]
+        {
+            use std::ptr::null;
+            use raw_window_handle::{HasRawWindowHandle, RawWindowHandle};
+            use icrate::AppKit::{NSViewHeightSizable, NSViewWidthSizable, NSVisualEffectBlendingModeBehindWindow, NSVisualEffectMaterialFullScreenUI, NSVisualEffectStateFollowsWindowActiveState, NSWindowBelow};
+            use icrate::Foundation::NSRect;
+            use objc2::{class, msg_send};
+            use objc2::runtime::NSObject;
+
+            let handle = self.winit_window().raw_window_handle();
+            let handle = match handle {
+                RawWindowHandle::AppKit(h) => h,
+                _ => return,
+            };
+
+            //let window = handle.ns_window as *mut NSObject;
+            let view = handle.ns_view as *mut NSObject;
+
+            let theme_view: *mut NSObject = unsafe { msg_send![view, superview] };
+
+            static BLURRED_VIEW: std::sync::OnceLock<usize> = std::sync::OnceLock::new();
```

Comment:
If this is static, what happens if this function is called on multiple windows?

Reply:
> If this is static, what happens if this function is called on multiple windows?

the blur effect would be transferred to the second window